apps/voting/app/src/hooks/useVotes.js

```diff
@@ -53,11 +51,12 @@ function useDecoratedVotes() {
       if (targetApp) {
         const { appAddress, icon, identifier, name } = targetApp
         executionTargetData = {
+          identifier,
           address: appAddress,
-          name,
+          // If the app name was not loaded, use the app's address
+          name: name || appAddress,
```

Comment:
Wondering if we should use the shortened version here?

Reply:
Yes, let's do that!